src/components/encounter/ImageWithBadge.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { View } from 'react-native';
+
+import { colors, device } from '../../config';
+import { User } from '../../types';
+import { Image } from '../Image';
+
+import { ImageBadge } from './ImageBadge';
+type Props = Pick<User, 'imageUri' | 'verified'>;
+export const ImageWithBadge = ({ imageUri, verified }: Props) => {
+  return (
+    <View style={styles.container}>
+      <View>
+        <View style={styles.circle}>
+          <Image source={{ uri: imageUri }} resizeMode="contain" />
+        </View>
+        <ImageBadge verified={verified} />
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  circle: {
```

Comment:
this circle styles duplicates the circle style from the encounter home screen, right? the outer view could be a `RoundedView` or `Circle` component or styled-component.

Reply:
The badges will be svgs instead of pngs, so i will need to rework the badges anyways. in the process of doing that i will check, if extracting the styling logic here is worthwhile (if the styles between badge, image and qr container share enough things in the end)

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/334